doc/appendix-fr-boi-tva-decl-30-10-30/cash-register-integration/cash-register-integration.md

```diff
@@ -104,6 +104,6 @@ The Data Collection Log consists of a localised set of information stored using
 
 Receipt journals are JSON exports of internal structured data related to receipts handled by the fiskaltrust.Middleware. For each receipt type, a journal is created and maintained.
 
-#### Action Journal
```

Comment:
Probably there's a reason but why only change action journal and not receipt journal above?